.github/workflows/gpt-commit-summarizer.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
GPT summary of [3ca98f](https://github.com/sebastianbarrozo/gpt-commit-summarizer/blob/c8c47408c482f791430fa4be8b1253005844ef70/.github/workflows/gpt-commit-summarizer.yml#3ca98f432722f3ed8391a7507f6fdea7d867ad36) - [07067d](https://github.com/sebastianbarrozo/gpt-commit-summarizer/blob/14bc7d58520d55fecaa099a6e45930d84b7be731/.github/workflows/gpt-commit-summarizer.yml#07067d2faa71acf26459dffc4248af017ba56dbc):
* Changed the used repository from KanHarI/gpt-commit-summarizer@master to sebastianbarrozo/gpt-commit-summarizer@master.